interface/modules/custom_modules/oe-module-claimrev-connect/src/Eligibility.php

```diff
@@ -0,0 +1,237 @@
+<?php 
+use OpenEMR\Modules\ClaimRevConnector\EligibilityData;
+use OpenEMR\Modules\ClaimRevConnector\EligibilityInquiryRequest;
+use OpenEMR\Modules\ClaimRevConnector\SubscriberPatientEligibilityRequest;
+use OpenEMR\Modules\ClaimRevConnector\EligibilityObjectCreator;
+use OpenEMR\Modules\ClaimRevConnector\ValueMapping;
+
+
+$insurance = EligibilityData::getInsuranceData($pid);
+?>
+
+<div class="row">
+    <div class="col">
+    <ul class="nav nav-tabs mb-2">
+        <?php  $classActive = "active"; $first="true"; foreach ($insurance as $row) { ?>
+            <li class="nav-item" role="presentation">
+                <a id="claimrev-ins-<?php echo(ucfirst($row['payer_responsibility']));?>-tab" aria-selected="<?php echo($first); ?>" class="nav-link <?php echo($classActive);?>"  data-toggle="tab" role="tab" href="#<?php echo(ucfirst($row['payer_responsibility']));?>"> <?php echo(ucfirst($row['payer_responsibility']));?>  </a>
+            </li>
+        <?php $first = "false"; $classActive=""; } ?>
+        
+    </ul>
+    <div class="tab-content">
+        <?php 
+        $classActive = "in active"; 
+        foreach ($insurance as $row) { 
+            $eligibilityCheck = EligibilityData::getEligibilityResult($pid,$row['payer_responsibility']);
+            ?>
+            <div id="<?php echo(ucfirst($row['payer_responsibility']));?>" class="tab-pane <?php echo($classActive);?>">
+                <div class="row">
+                    <div class="col-2">
+                        <form method="post" action="<?=$_SERVER['PHP_SELF'];?>">
```

Comment:
rec not using php shortcuts; `instead use <?php echo`

rec not using $_SERVER for links since could be potentially modified by bad actor and instead hard-code the path either via relative link or absolute link incorporating $GLOBALS['fileroot']

Reply:
I think this is fixed now